eessi_bot_event_handler.py

```diff
@@ -692,7 +697,7 @@ def main():
     opts = event_handler_parse()
 
     # config is read and checked for settings to raise an exception early when the event_handler starts.
-    if config.check_required_cfg_settings(REQUIRED_CONFIG):
+    if config.check_cfg_settings(REQUIRED_CONFIG):
```

Comment:
`REQUIRED_CONFIG` is defined in this file, but the new `FORBIDDEN_CONFIG` is added in `config.py`. Unless I'm overlooking a good reason to have them in different files, I'd say that they belong in the same file? And probably `config.py` makes more sense? And then the function can simply be called without arguments.

Reply:
The `event_handler` and `job_manager` each have a _different_ `REQUIRED_CONFIG`, which is why they are defined there instead of in a central place like `config.py`. The `FORBIDDEN_CONFIG` is central: it lists the items that should simply no longer be defined in the config since they are not used by _any script_ anymore.

Reply:
Okay, I overlooked that, that does actually make sense and is a valid reason to have them split.